src/components/layouts/Header.tsx

```diff
@@ -93,7 +93,10 @@ export default function Header() {
         )}
       </header>
       <AccountModal />
-      <RecommendsModal isOpen={isOpenRecommends} onClose={onCloseRecommends} />
+      {/* TODO: summonerSearchBar 활성 시 제거 */}
+      {isAuthenticated && myInfoData !== undefined && (
+        <RecommendsModal isOpen={isOpenRecommends} onClose={onCloseRecommends} />
+      )}
```

Comment:
```suggestion
      {isAuthenticated && (
        <RecommendsModal isOpen={isOpenRecommends} onClose={onCloseRecommends} />
      )}
```
여기선 `myInfoData`가 사용되지 않으므로 `isAuthenticated`만 확인해도 상관 없을 것 같습니다.